### ISSUE TITLE
Updates to Staged CKAN Fail

### ISSUE THREAD

**Issue by techman83** [KSP-CKAN/NetKAN-bot#83]
@HebaruSan This is the error from the bot
```
error: could not apply 2cb019282...  "NetKAN generated mods - MechJeb2-2.8.2.0"
hint: after resolving the conflicts, mark the corrected paths
hint: with 'git add <paths>' or 'git rm <paths>'
hint: and commit the result with 'git commit'
```
This is the git status
```
On branch staging
You are currently cherry-picking commit 2cb019282.
  (fix conflicts and run "git cherry-pick --continue")
  (use "git cherry-pick --abort" to cancel the cherry-pick operation)

Unmerged paths:
  (use "git add/rm <file>..." as appropriate to mark resolution)

	deleted by us:   MechJeb2/MechJeb2-2.8.2.0.ckan

no changes added to commit (use "git add" and/or "git commit -a")
netkan@ckan-bot:~/NetKAN/CKAN-meta$ git show 2cb019282
commit 2cb0192826ac3ef30dd63703a3da238f360d9e46 (ouZRikgt)
Author: NetKAN inflator Robot <netkan-bot@ksp-ckan.space>
Date:   Tue Feb 5 06:08:57 2019 +0000

     "NetKAN generated mods - MechJeb2-2.8.2.0"

diff --git a/MechJeb2/MechJeb2-2.8.2.0.ckan b/MechJeb2/MechJeb2-2.8.2.0.ckan
index 97e00595d..fd2c67a86 100644
--- a/MechJeb2/MechJeb2-2.8.2.0.ckan
+++ b/MechJeb2/MechJeb2-2.8.2.0.ckan
@@ -11,7 +11,7 @@
         "ci": "https://ksp.sarbian.com/jenkins/job/MechJeb2-Release/"
     },
     "version": "2.8.2.0",
-    "ksp_version": "1.5",
+    "ksp_version": "1.6",
     "install": [
         {
             "find": "MechJeb2",
```
What is happening is that MechJeb2-2.8.2.0 doesn't exist on the staging branch, and we're trying to apply a cherry-pick that is only the change to said file.

When I wrote this I didn't test for what happened when a staged CKAN needed changes. I also don't think we could add multiple meta data sources in the client, so the staging branch is probably superfluous to our needs now. Authors with staging enabled can just add their own branch as a repo.

**Comment by HebaruSan** [comment]
Fixed in #84

**Comment by techman83** [comment]
In theory :smile:
